lib/foreman/renderer/configuration.rb

```diff
@@ -42,7 +42,11 @@ class Configuration
         :number_with_delimiter,
         :number_with_precision,
         :number_to_human_size,
-        :gem_version_compare
+        :gem_version_compare,
+        :parse_yaml,
+        :parse_json,
+        :to_json,
+        :to_yaml
```

Comment:
This is the third PR today I see changing the `gem_version_compare` line. Let's use trailing commas
```suggestion
        :to_yaml,
```

Reply:
Yeah I hit this the other day. Honestly, I hate this cop. Enforcing either way is evil, I tend to keep the comma these days but this should not be a failure.